nautobot/extras/filters.py

```diff
@@ -496,7 +496,7 @@ class GitRepositoryFilterSet(NautobotFilterSet):
 
     class Meta:
         model = GitRepository
-        fields = ["id", "name", "slug", "remote_url", "branch"]
+        fields = ["id", "name", "slug", "remote_url", "branch", "provided_contents"]
```

Comment:
Ugh, we have no testing of `GitRepositoryFilterSet` in `nautobot/extras/tests/test_filters.py`.  :-( Would you be willing to take a stab at adding a few tests?

Reply:
Will do

Reply:
@glennmatthews done :-)